osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/mapping/WsInstallationDeviceToDeviceConverter.java

```diff
@@ -12,6 +12,7 @@
 import java.util.Optional;
 
 import org.opensmartgridplatform.adapter.ws.schema.core.deviceinstallation.Device;
+import org.opensmartgridplatform.adapter.ws.schema.core.devicemanagement.Manufacturer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.opensmartgridplatform.adapter.ws.schema.core.devicemanagement.Manufacturer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)